packages/frontend/src/components/EditorLayout/index.tsx

```diff
@@ -154,24 +153,27 @@ export default function EditorLayout() {
             )}
           </Flex>
 
-          <Button
-            as={Link}
-            to={URLS.GUIDE_LINK}
-            colorScheme="secondary"
-            target="_blank"
-            variant="link"
-          >
-            Guide
-          </Button>
+          <TouchableTooltip label="Guide" aria-label="guide tooltip">
+            <IconButton
+              as={Link}
+              to={URLS.GUIDE_LINK}
+              target="_blank"
+              variant="clear"
+              aria-label="guide"
+              icon={<BiInfoCircle />}
+              colorScheme="secondary"
+            />
+          </TouchableTooltip>
 
           <TouchableTooltip label="Settings" aria-label="settings tooltip">
             <IconButton
               as={Link}
               to={URLS.FLOW_EDITOR_NOTIFICATIONS(flowId)}
-              variant="outline"
+              variant="clear"
               aria-label="settings"
               icon={<BiCog />}
-            ></IconButton>
+              colorScheme="secondary"
+            />
```

Comment:
standardising the guide and settings to icons look good! what do you think of adding some contrast on hover?
```suggestion
              _hover={{
                color: 'primary.500',
                bg: 'interaction.muted.main.hover',
              }}
```

Reply:
done thanks!